swag_auth/bitbucket/connectors.py

```diff
@@ -45,7 +29,7 @@ def get_user_repo(self, repo_name):
         :param repo_name:
         :return:
         """
-        return
+        return repo_name
```

Comment:
What is this? This method should return a content of the file, not the repo name...